packages/dashboard/src/testing/testing-ground/testing-ground.tsx

```diff
@@ -43,6 +160,57 @@ export class TestingGround {
     this.stretchToFit = (e as any).target.checked;
   };
 
+  addAnnotation = (e: Event) => {
+    e.preventDefault();
+    const original = this.dashboardConfiguration;
+    const XANNOTATION: XAnnotation = {
+      color: 'red',
```

Comment:
Should we make an enum for annotation colors so we don't have magic strings?

Reply:
The color picker in the annotation controls allows for hex representation of colors to be stored in 'color'